src/UI/Utils/Action.elm

```diff
@@ -0,0 +1,60 @@
+module UI.Utils.Action exposing
```

Comment:
feels a little weird having this under utils. What about something like `UI.Config.Action`, `UI.Data.Action` or even just `UI.Action`?